server/api/user_handler.py

```diff
@@ -63,6 +63,11 @@ def put(self):
         current_userid = get_jwt_identity()
         user = User.get_one_user(current_userid)
 
+        if req_body.get('email'):
+            return custom_json_response({
+                "error": "Cannot use this route to update email address."
+            }, 403)
+
```

Comment:
Nice job catching this. Would it make sense to just use dump_only in the schema for email?

Reply:
dump_only would make it a read-only. its a good suggestion but wouldn't work in practice. It would prevent PUT from changing the email but it would also prevent  POST requests for new users from submitting emails.